### PR TITLE
dash/HighchartsTypes

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed `d.ts` files for `HighchartsTypes`.

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

File: ts/Dashboards/Plugins/HighchartsComponent.ts [hunk 1]
```diff
@@ -751,30 +735,29 @@ class HighchartsComponent extends Component {
...
             }
         }
 
         if (typeof charter.chart !== 'function') {
             throw new Error('Chart constructor not found');
         }
 
-        this.chart = charter.chart(this.chartContainer, this.chartOptions);
-        return this.chart;
+        return this.chart as any;
```
Comment (bre1470) [inline ts/Dashboards/Plugins/HighchartsComponent.ts:760]: I do not like the use of `any`. The previous one worked without it. Does it lack declarations in HighchartsTypes?

File: ts/Dashboards/Plugins/KPIComponent.ts [hunk 2]
```diff
@@ -341,7 +340,7 @@ class KPIComponent extends Component {
         super.render();
         this.updateElements();
 
-        const charter = KPIComponent.charter;
+        const charter = KPIComponent.charter as any;
```
Comment (bre1470) [inline ts/Dashboards/Plugins/KPIComponent.ts:343]: This probably also looks like in need of a type fix.

File: ts/Dashboards/Plugins/NavigatorComponent.ts [hunk 3]
```diff
@@ -456,8 +456,8 @@ class NavigatorComponent extends Component {
 
         const charter = (
             NavigatorComponent.charter ||
-            Globals.win.Highcharts as H
-        );
+            Globals.win.Highcharts
+        ) as any;
```
Comment (bre1470) [inline ts/Dashboards/Plugins/NavigatorComponent.ts:460]: Why?

**Review by bre1470 — CHANGES_REQUESTED** [review]
Looks good, except `any`s.

**Review by bre1470 — APPROVED** [review]
Awesome! 👍

**Pull request merged**